### DISCUSSION TITLE
How to use group field with Elementor Pro (works with text and text area)

### DISCUSSION THREAD

**Discussion by birdbrainsolutions** [elementor/elementor#15481] [Developers Community]
Hi guys,

I recently created a group field with ACF however it wasn't showing in Elementor Pro. 

I figured a workaround for it, however it only works with one sub field at a time, but atleast this way you can use the group field in the backend. 

To insert the sub field, use the shortcode widget in Elementor and insert the ACF shortcode as follows:

Field Name: example

Group Sub Field Name: sample

[acf field="example_sample"] and use custom css to style it. 

Repeat the process for all sub fields. You have to use the group field name with the sub field name or else it won't work. 

Hope this helps atleast someone.

**Comment by digitalinksol** [comment]
Hoping the ability to use ACF Group / Subfields makes it into a future version of Elementor.

**Comment by Aviad-Herman** [comment]
Hi,
Currently we do not support ACF group fields but I will convert this thread to a discussion for future reference.

**Comment by fanchon-adverti** [comment]
Omg, thank you very much !
